### PR TITLE
Updated readme.md File

### PR DESCRIPTION
**Pull request by CaptainNayan199**

### CODE REVIEW

File: README.md
Comment (CaptainNayan199) [inline README.md:1]: I noticed that the NumPy readme.md hadn't been updated recently, so I decided to add some new features that I found when working with NumPy.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ https://doi.org/10.1038/s41586-020-2649-2)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/numpy/numpy/badge)](https://api.securityscorecards.dev/projects/github.com/numpy/numpy)
 
 
-NumPy is the fundamental package for scientific computing with Python.
+NumPy is a critical package or module for doing scientific calculations with Python.
```
Comment (charris) [inline README.md:20]: I don't think this is an improvement.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ It provides:
 - sophisticated (broadcasting) functions
 - tools for integrating C/C++ and Fortran code
 - useful linear algebra, Fourier transform, and random number capabilities
+- can be easily integrated with other libraries
+- efficient memory usage
+- compatible with other language
+
```
Comment (charris) [inline README.md:38]: Not convinced this adds much.

File: README.md [hunk 3]
```diff
@@ -32,6 +32,14 @@ It provides:
...
+Installing:
+
+- NumPy can be installed on your native machine by the use of `pip` command in terminal
+- paste this command into your terminal : `pip install NumPy`
```
Comment (charris) [inline README.md:43]: Using pip like that is dangerous, instead `python -m pip install numpy`  should be used.

**Comment by seberg** [comment]
I am going to close this.  Overall some good thoughts but honestly not an improvement.  There are some smaller ones I see possible: improve the docs link for example.  Maybe also add an install/how to get link to numpy.org (which IIRC has instructions).

File: README.md (hunk 3 above)
Comment (CaptainNayan199) [reply README.md:43]: Thank you for the suggestion Sir! Much appreciated.

**Pull request closed without merging**